README.md

```diff
@@ -16,84 +16,84 @@ We should be as specific and thorough as possible when defining our style, testi
 
 #### 1. Names of internal functions in a library should not have an underscore prefix.
 
-The style guide states
-
-> Underscore Prefix for Non-external Functions and Variables
-
-One of the motivations for this rule is that it is a helpful visual clue.
+YES:
 
-> Leading underscores allow you to immediately recognize the intent of such functions...
+```solidity
+Library.function()
+```
 
-We agree that a leading underscore is a useful visual clue, and this is why we oppose using them for internal library functions that can be called from other contracts. Visually, it looks wrong.
+NO:
 
 ```solidity
 Library._function()
 ```
 
-or
+If a function should never be called from another contract, it should be marked private and its name should have a leading underscore.
+
+#### Justification
+
+The style guide states `Underscore Prefix for Non-external Functions and Variables`. One motivation for this rule is providing a helpful visual clue:
+
+> Leading underscores allow you to immediately recognize the intent of such functions...
+
+We agree that a leading underscore is a useful visual clue, and this is why we oppose using them for internal library functions that can be called from other contracts. Visually, it looks wrong.
 
 ```solidity
 using Library for bytes
 bytes._function()
 ```
 
-Note, we cannot remedy this by insisting on the use public functions. Whether a library functions are internal or external has important implications. From the [Solidity documentation](https://docs.soliditylang.org/en/latest/contracts.html#libraries)
+We could remedy this by insisting on the use public functions. However, developers may prefer internal functions because they are more gas efficient to call, due to how libraries are compiled in Solidity:
```

Comment:
imo is confusing to say "we could" remedy. Maybe like "We should not remedy this by ..."